src/x11/x.c

```diff
@@ -81,18 +81,18 @@ static void x_win_move(window_x11 *win, int x, int y, int width, int height)
         }
 }
 
-void x_win_round_corners(window_x11 *win, const int rad)
+static void x_win_round_corners(window_x11 *win, const int rad)
 {
         const int width = win->dim.w;
         const int height = win->dim.h;
+        const int dia = 2 * rad;
+        const int degrees = 64;
```

Comment:
I'd add a comment here to explain *why* 64, or just point to the docs.

Reply:
Err, this is [because of the docs](https://tronche.com/gui/x/xlib/graphics/filling-areas/XFillArc.html) :tm:.

Reply:
I'm aware, but someone looking at it a few months or years in the future is going to be really confused at this random value.

Reply:
Good point. It definitely warrants a comment.